panimg/models.py

```diff
@@ -51,8 +51,8 @@ class PanImg:
     voxel_depth_mm: Optional[float]
     timepoints: Optional[int]
     resolution_levels: Optional[int]
-    window_center: Optional[float]
-    window_width: Optional[float]
+    window_center: Union[None, float, Tuple[float, ...]]
+    window_width: Union[None, float, Tuple[float, ...]]
```

Comment:
Unfortunately you cannot change the datatype here without a schema migration on grand challenge and a data migration there. The models are only for what we index. I would suggest only returning the first value pair that is found, ultimately these data are not used as CIRRUS gets its information from the MHA, not the API.

Reply:
Ah, hadn't traced the object through to GC. Will implement your suggestion.